packages/dataviews/src/types.ts

```diff
@@ -498,6 +498,8 @@ export interface ViewBaseProps< Item > {
 	onChangeSelection: SetSelection;
 	selection: string[];
 	setOpenedFilter: ( fieldId: string ) => void;
+	onItemClick: ( item: Item ) => void;
```

Comment:
I'd prefer if we go with `onClickItem`, like the docs say — this would follow the existing naming schema (`onChangeSelection`, on + verb + noun).

Reply:
Fixed with 419f3fa7d2ee0da4c855087d7cc74ef88030cc4a